help_to_heat/portal/models.py

```diff
@@ -165,3 +165,53 @@ class ScottishEpcRating(utils.TimeStampedModel):
     uprn = models.CharField(max_length=12, primary_key=True)
     rating = models.CharField(max_length=32, choices=epc_rating_choices)
     date = models.DateField(blank=True, null=True)
+    property_type = models.CharField(max_length=32, null=True)
+    address1 = models.CharField(max_length=64, null=True)
+    address2 = models.CharField(max_length=64, null=True)
+    address3 = models.CharField(max_length=64, null=True)
```

Comment:
I would probably go longer with these too

Reply:
I think 512 should be sufficient.

I've found some discussion online that the longest address in the royal mail PAF service is about 370 characters. (But I think that might encompass all three lines.)